### ISSUE TITLE
unresolved import `deadpool_redis::redis::Config`

### ISSUE THREAD

**Issue by cryply** [bikeshedder/deadpool#120]
Hey 

adding to my Cargo.toml 
`deadpool-redis = { version = "0.8", features = ["config"] }`

and main.rs
`use deadpool_redis::redis::{cmd, Config, FromRedisValue};`

generates```

error[E0432]: unresolved import `deadpool_redis::redis::Config`
 --> src/symhan.rs:2:34
  |
2 | use deadpool_redis::redis::{cmd, Config, FromRedisValue};
  |                                  ^^^^^^ no `Config` in `redis`
```

**Comment by bikeshedder** [comment]
The `Config` struct is part of `deadpool_redis` crate itself:

```rust
 use deadpool_redis::Config;
 use deadpool_redis::redis::{cmd, Config, FromRedisValue};
```

 `deadpool_redis::redis` is just the re-exported `redis` crate:
https://docs.rs/deadpool-redis/0.8.1/deadpool_redis/#reexports

**Comment by bikeshedder** [comment]
Oh, I see... the example in the README is wrong. I need to fix that.

**Comment by bikeshedder** [comment]
This was fixed by #127
